stack-nightly.yaml.lock

```diff
@@ -3,10 +3,52 @@
 # For more information, please see the documentation at:
 #   https://docs.haskellstack.org/en/stable/lock_files
 
-packages: []
+packages:
+- completed:
+    hackage: crypton-1.0.0@sha256:637e58581978c84ef1288d14fa9cac1d2905ef60e319924293bc11250aca882d,14527
+    pantry-tree:
+      sha256: 4b5e5511567c0fe735a224cb8b2b278e1caa79344f2940d030d169e69b1b81e1
+      size: 23275
+  original:
+    hackage: crypton-1.0.0
+- completed:
+    hackage: amazonka-2.0@sha256:d401ac9094bb331d1bfb9be879d9007ca6bc94085ab745de5aa827128ad7917c,3522
+    pantry-tree:
+      sha256: b9fa0aaccff472ac93200d505678fb560ce15a9e409bdff61e1a3e08ed89d344
+      size: 1528
+  original:
+    hackage: amazonka-2.0
+- completed:
+    hackage: amazonka-core-2.0@sha256:b60507a2791d05ce6fb69544a854c0ee0ec794908400aebe9adcacedf9cd4e17,4585
+    pantry-tree:
+      sha256: 60d4c21ff717e3cfa3fc78812c6d1032e22c27d9427d2ab42f649a6cc0b70d0d
+      size: 3222
+  original:
+    hackage: amazonka-core-2.0
+- completed:
+    hackage: amazonka-sso-2.0@sha256:dfb9232d66fac2a912f63aa5ddaf56ea495649cc3ab451b5102f8ec579dedc4f,3012
+    pantry-tree:
+      sha256: 1018821c55d328ad72e532267bc27528353fc226ad43817a9e8ea91509b9b7a9
+      size: 1817
+  original:
+    hackage: amazonka-sso-2.0
+- completed:
+    hackage: amazonka-sts-2.0@sha256:949968463bde023a6b8035527b5da5585da8e3c36f0366eedf663aac2b9d5d12,3226
+    pantry-tree:
+      sha256: b761d4c67118c25f4711c183b4983903a3a759d5802b91795f603cb0555bfb0e
+      size: 2880
+  original:
+    hackage: amazonka-sts-2.0
+- completed:
+    hackage: amazonka-s3-2.0@sha256:1d93bec43e0fc9bc3a57c43aeeb010c74a60afd3af5acd077731aad923f99802,13870
+    pantry-tree:
+      sha256: 0b88c93134287ca43f169d2163acce45700bab1ea81e97f11de4d4330db7c9fa
+      size: 38690
+  original:
+    hackage: amazonka-s3-2.0
 snapshots:
 - completed:
-    sha256: a4e61f552492f4f0a14246c054144a8b6b455c67c9e99fddf891d009baa65604
-    size: 667656
-    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/nightly/2023/9/9.yaml
-  original: nightly-2023-09-09
+    sha256: b69a06c70e1092e2aa60ce69a4877c6181fbe53dfb7ea041cb98d97617d6007f
+    size: 716491
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/nightly/2023/12/26.yaml
+  original: nightly-2023-12-26
```

Comment:
This seems wrong :thinking:

Reply:
Oops, I thought the revert would have fixed that too. I ran `stack --stack-yaml=stack-nightly.yaml build --fast --pedantic --test` it should be right now, though it fails